Ladybird/AppKit/UI/Tab.h

```diff
@@ -9,6 +9,7 @@
 #include <AK/Types.h>
 
 #import <Cocoa/Cocoa.h>
+#import <UI/Inspector.h>
```

Comment:
You can forward-declare Inspector here instead of including it's full definition, like we do with LadybirdWebView